platform/web/export/export_plugin.cpp

```diff
@@ -153,7 +153,7 @@ void EditorExportPlatformWeb::_fix_html(Vector<uint8_t> &p_html, const Ref<Edito
 	const String custom_head_include = p_preset->get("html/head_include");
 	HashMap<String, String> replaces;
 	replaces["$GODOT_URL"] = p_name + ".js";
-	replaces["$GODOT_PROJECT_NAME"] = ProjectSettings::get_singleton()->get_setting("application/config/name");
+	replaces["$GODOT_PROJECT_NAME"] = ProjectSettings::get_singleton()->get_setting_with_override("application/config/name");
```

Comment:
Can't we use `GLOBAL_GET` for those for consistency in the codebase? Ideally nothing would use `get_setting_with_override` directly aside from `GLOBAL_GET` itself.

Reply:
Yeah I don't see why not